test/test-applications/integrations/Samples.Msmq/Samples.Msmq.csproj

```diff
@@ -0,0 +1,16 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <!-- override to remove core -->
+    <TargetFrameworks>net452;net461</TargetFrameworks>
+    <OutputType>Exe</OutputType>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Debug|net452|x64'">
+    <WarningLevel>4</WarningLevel>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Debug|net452|x86'">
+    <WarningLevel>4</WarningLevel>
+  </PropertyGroup>
```

Comment:
I'm guessing VS added this. We should probably remove these lines since they only apply to `Debug` and `net452`. If we want `<WarningLevel>4</WarningLevel>` for all configurations, we can add it to the `<PropertyGroup>` above.